pycbc/events/stat.py

```diff
@@ -413,6 +434,16 @@ def coinc(self, s0, s1, slide, step):
         # scale to resemble network SNR
         return cstat / (2.**0.5)
 
+class PhaseTDExpFitMultiStatistic(PhaseTDExpFitStatistic):
```

Comment:
hmm. don't know where 'Multi' fits in here .. maybe it would be better to rename more stuff to clear up the distinction between the newsnr-based combined stat and the newsnr+sg one 
eg phasetd_exp_fit_stat -> phasetd_exp_fit_newsnr 
then new thing is phasetd_expfit_newsnr_sg[veto]